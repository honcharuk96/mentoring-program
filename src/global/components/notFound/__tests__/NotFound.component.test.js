import React from 'react'
import NotFound from '../NotFound.component';
import ShallowRenderer from 'react-test-renderer/shallow';

const shallow = new ShallowRenderer();
jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useLocation: () => ({
        pathname: 'localhost:3000/example/path',
    }),
}));

describe('<NotFound />', () => {
    const notFound =  shallow.render(<NotFound/>);
    it('should render ExampleComponent', () => {
         expect(notFound).toMatchSnapshot();
         expect(JSON.stringify(notFound)).toMatch('localhost:3000/example/path')

    });
});
