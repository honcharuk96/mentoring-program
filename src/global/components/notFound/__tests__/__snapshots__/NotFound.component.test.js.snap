// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render ExampleComponent 1`] = `
<div>
  <h3>
    No match for 
    <code>
      localhost:3000/example/path
    </code>
  </h3>
</div>
`;
